crates/bin/src/ui.rs

```diff
@@ -108,6 +108,10 @@ pub fn logging(args: &Args) {
     log_config.add_filter_ignore("hyper".to_string());
     log_config.add_filter_ignore("reqwest".to_string());
     log_config.add_filter_ignore("rustls".to_string());
+
+    #[cfg(feature = "trust-dns")]
+    log_config.add_filter_ignore("trust_dns".to_string());
```

Comment:
honestly I think instead of adding ignores we should do the inverse and filter to _just_ the `binstall` logs, and only if the `log level != trace`.

Reply:
That's a brilliant idea, I will do that later.

Reply:
Done!